website/tailwind.config.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
       padding: "1rem",
     },
   },
+  important: true,
```

Comment:
Tired of battling with docusaurus and others styling, gonna clobber with `important` here, which is somewhat common in these sort of setups.